kratos/sources/kernel.cpp

```diff
@@ -25,7 +25,7 @@ Kernel::Kernel() : mpKratosCoreApplication(Kratos::make_shared<KratosApplication
                     << "           Multi-Physics " << KRATOS_VERSION << std::endl;
 
     if (!IsImported("KratosMultiphysics")) {
-        mpKratosCoreApplication->RegisterVariables();
+      //mpKratosCoreApplication->RegisterVariables();
```

Comment:
please remove code, do not comment
This is really confusing if someone else looks at this (thinking if this is intentional or debugging leftovers)
Esp in those parts of the code

The old state can always be recovered through the git-history

Reply:
Thanks